frontend/src/app/users/users-profile/users-profile.component.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { ProjectService } from 'src/app/projects/service/project.service';
+import { User, UserService } from 'src/app/services/user/user.service';
+
+@Component({
+  selector: 'app-users-profile',
+  templateUrl: './users-profile.component.html',
+  styleUrls: ['./users-profile.component.css'],
+})
+export class UsersProfileComponent implements OnInit {
+  userName: string | undefined;
+  userId: number | undefined;
+  currentUser: User | undefined;
+  constructor(
+    public userService: UserService,
+    public projectService: ProjectService,
+    private route: ActivatedRoute,
+  ) {}
+
+  ngOnInit() {
+    this.route.queryParams.subscribe((params) => {
+      this.userId = params.userId;
+      this.userName = params.userName;
```

Comment:
Please use the consistent URI schema for the frontend, more information here: https://dsd-dbs.github.io/capella-collab-manager/development/frontend/routes/

==> It should be `/user/<id>`, no need for query params here

Also, I'd remove `params.userName`. We shouldn't allow the user to provide the username for a specific user id. For example, the user could open the page with the params `?userId=1&userName=test` and it would display the username test (even though the user has a different name in the backend).

Only the ID is required and can be used to fetch the user from the backend.

Reply:
Once you fetch the user from the backend, you can also save the user in the user.service like described in the development docs: https://dsd-dbs.github.io/capella-collab-manager/development/frontend/code-style/#retriving-resources

Reply:
fair - I totally see your point 
the problem that I was trying to solve was: that as a non-admin user I am not allowed to fetch users from the backend... which also makes sense in order to not just ask for ids and get users back thus I wanted to use the username as long as we anyways have it 
do you have another solution for that problem?

Reply:
Don't think this is resolved... 
The URL is still `user?userId=17`... 
Will mark it as unresolved.

Reply:
should be fixed now